ui/ui-enterprise/pages/destination/[id]/data.tsx

```diff
@@ -77,7 +80,7 @@ export default function Page(props) {
 
```

Comment:
This is a lot of logic to have in the UI.  It's my fault for setting it up this way.

I think the path forward is to have one API action (eg: `destination:edit`, `PUT /destination/id`) that can also accept a `trackedGroupId`, and we handle this logic in the Action.  This way the form could just submit whatever it has.

* if `trackedGroupId === destination.groupId` do nothing
* if `trackedGroupId === null || ""` call `desintaion.untrack()`
* if `trackedGroupId !== destination.groupId` call `destinatio.track(trackedGroupId)`

Reply:
This may also mean we can remove the Actions for destination:track and destination:untrack

Reply:
I think it's also confusing that the save button on this page /sometimes/ also means "re-export everyone" (that's the case that does `await execApi("post", `/destination/${id}/export`, { force: true });`.  That should probably be moved to it's own button called "re-export group members" or similar.